### PR TITLE
fix(tooltip): Edge styles fix, arrow styles update

### PR DESCRIPTION
**Pull request by LeonVay**
@Fost , @lskramarov , @mikeozornin please take a look.
@mikeozornin  please note: arrow height and width were changed to match 16px width on template between two extreme points.

### CODE REVIEW

File: packages/mosaic/tooltip/tooltip.scss [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 $zindex-tooltip: 1060;
 $tooltip-max-width: 240px;
-$border-radius-base: 3px;
-$tooltip-arrow-width: 8px;
-$tooltip-distance: $tooltip-arrow-width + 3px;
+$border-radius-base: 4px;
```
Comment (lskramarov) [inline packages/mosaic/tooltip/tooltip.scss]: why did you change it ?
on sketch border-radius: 3px;
https://app.zeplin.io/project/5b902f859dd1892babf00694/screen/5afb0c9f9a9ea13f3664e95f

File: packages/mosaic/tooltip/tooltip.scss [hunk 2]
```diff
@@ -57,36 +57,31 @@ $tooltip-distance: $tooltip-arrow-width + 3px;
 
 .mc-tooltip-arrow {
     position: absolute;
-    width: 0;
-    height: 0;
-    border-color: transparent;
-    border-style: solid;
+    width: 11.3px;
```
Comment (lskramarov) [inline packages/mosaic/tooltip/tooltip.scss]: why do you use float number ?

**Comment by lskramarov** [comment]
@LeonVay UP

File: packages/mosaic/tooltip/tooltip.scss (hunk 2 above)
Comment (LeonVay) [reply packages/mosaic/tooltip/tooltip.scss]: On template the width between two corner points of the arrow is 16px. I am setting an arrow as a square and rotating it.

File: packages/mosaic/tooltip/tooltip.scss (hunk 1 above)
Comment (LeonVay) [reply packages/mosaic/tooltip/tooltip.scss]: Right, it's my mistake.

File: packages/mosaic/tooltip/tooltip.scss (hunk 2 above)
Comment (lskramarov) [reply packages/mosaic/tooltip/tooltip.scss]: It didn't solve problem, now width (or height) 15px:
image

and if you zoom example you will see:
image

I suggest set width and height to 12px:
image
and
image

Comment (LeonVay) [reply packages/mosaic/tooltip/tooltip.scss]: Ok, thanks. Just a minute and I will update PR.

Comment (LeonVay) [reply packages/mosaic/tooltip/tooltip.scss]: Updated
